storage-proofs/src/sector_id.rs

```diff
@@ -0,0 +1,30 @@
+use byteorder::ByteOrder;
+
+#[derive(Debug, Clone, Copy, PartialOrd, PartialEq, Eq)]
+pub struct SectorId(u64);
```

Comment:
this should be `#[repr(C)]` otherwise things will be painful in the ffi boundary

Reply:
sorry, spoke too soon `repr(transparent)` isl likely what we want

Reply:
https://github.com/rust-lang/rfcs/blob/master/text/1758-repr-transparent.md

Reply:
I would not expose `SectorId` through the FFI (and rust-fil-proofs doesn't know about FFIs).

Instead, the crates which do FFI things would [create their own FFI-compatible representation of `SectorId`](https://github.com/filecoin-project/rust-fil-sector-builder/blob/f5ac51b7739041d63a9345f0f0d13ab77762d91a/sector-builder-ffi/src/api.rs#L16).